docs/guides/integration/fastapi.md

```diff
@@ -0,0 +1,136 @@
+# Using uv with FastAPI
+
+[FastAPI](https://github.com/fastapi/fastapi) is modern, high-performance Python web framework. You
+can use uv to manage your FastAPI project, including installing dependencies, managing environments,
+running FastAPI applications, and more.
+
+!!! note
+
+    You can view the source code for this guide in the [uv-fastapi-example](https://github.com/astral-sh/uv-fastapi-example) repository.
+
+As an example, consider the sample application defined in the
+[FastAPI documentation](https://fastapi.tiangolo.com/tutorial/bigger-applications/), structured as
+follows:
+
+```plaintext
+.
+├── app
+│   ├── __init__.py
+│   ├── main.py
+│   ├── dependencies.py
+│   └── routers
+│   │   ├── __init__.py
+│   │   ├── items.py
+│   │   └── users.py
+│   └── internal
+│       ├── __init__.py
+│       └── admin.py
+```
+
+To migrate this project to uv, add a `pyproject.toml` file to the root directory of the project, as
+a sibling to the `app` directory.
+
+```plantuml
+.
+├── pyproject.toml
+├── app
+│   ├── __init__.py
+│   ├── main.py
+│   ├── dependencies.py
+│   └── routers
+│   │   ├── __init__.py
+│   │   ├── items.py
+│   │   └── users.py
+│   └── internal
+│       ├── __init__.py
+│       └── admin.py
+```
+
+The contents of the `pyproject.toml` file should look something like this:
+
+```toml title="pyproject.toml"
+[project]
+name = "app"
+version = "0.1.0"
+description = "FastAPI project"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "fastapi",
```

Comment:
```suggestion
    "fastapi[standard]",
```

Reply:
extras/optional dependencies are specified in `[project.optional-dependencies]`, more info [here](https://docs.astral.sh/uv/concepts/dependencies/#optional-dependencies)